lib/jira/core.rb

```diff
@@ -61,17 +60,10 @@ def ticket?(ticket, verbose=true)
       ### Relevant Paths
 
       #
-      # @return [String] path to .jira-url file
+      # @return [String] path to .jira-cli file
       #
-      def url_path
-        @url_path ||= self.root_path + "/.jira-url"
-      end
-
-      #
-      # @return [String] path to .jira-auth file
-      #
-      def auth_path
-        @auth_path ||= self.root_path + "/.jira-auth"
+      def cli_path
+        @cli_path ||= self.root_path + "/.jira-cli"
```

Comment:
Sounds good. Thinking about it more, we likely are going to want to track the current sprint / current project as well. I'm not a huge fan of having a dependency to a seemingly unsupported project though ( https://github.com/TwP/inifile ). We should consider other alternatives, but fine with this for now